### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thundergreen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ First matrix panel start. When matrix panel publish message "up" in wled/status
     - service: mqtt.publish
       data:
         topic: wled/scrolleffect
-        payload_template: "9"
+        payload_template: "PACMAN"
```
Comment (widapro) [inline README.md:160]: Thank you for your attention!

File: README.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ First matrix panel start. When matrix panel publish message "up" in wled/status
     - service: mqtt.publish
       data:
         topic: wled/scrolleffect_without_exit
-        payload_template: "26"
+        payload_template: "PACMAN"
```
Comment (widapro) [inline README.md:174]: Thank you for your attention!

File: README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ You can paint icons whatever you want using this online tool: https://pjrp.githu
 
 Feel free to add new icons and update the existing `wLedFont.h` in the repository.
 <p align="center">
-<img width="800" height="259" title="wLed icons map" alt="wLed icons map" src="https://raw.githubusercontent.com/widapro/pixel_led_mqtt_panel/master/img/icons_map.jpg">
+<img width="800" height="259" title="wLed icons map" alt="wLed icons map" src="https://user-images.githubusercontent.com/13320319/74108108-fd881700-4b76-11ea-963a-20427e6bc831.png">
```
Comment (widapro) [inline README.md]: Could you remove these changes, I prefer to have a screenshot as a file in the repository. I had updated icons map in the repository.

**Review by widapro — CHANGES_REQUESTED** [review]

**Review by thundergreen — COMMENTED** [review]
You are welcome. I love this project and wann push it :)

**Comment by widapro** [comment]
cool, thx!

**Pull request merged**